### PR TITLE
Add patch to update PD Org price_plans

### LINKED ISSUE #258: Patch: update all pipedrive orgs with company.price_plan
Need to update all the pd Orgs with the `price_plan` on Company

### PR DESCRIPTION
**Pull request by PrenSJ2**
Close #258

# Description
get all Companies with a price_plan

post object to pd to update the Org

# Testing
- setup
- create new data field for `price_plan`﻿
- create new custom field for `price_plan`
- run patch
- see Org now has new populated field

### CODE REVIEW

**Comment by PrenSJ2** [comment]
image

**Review by DanJezeph15 — APPROVED** [review]
lgtm!

File: patch.py [hunk 1]
```diff
@@ -92,6 +92,24 @@ async def update_pd_org_price_plans():
 
     print(f'Updated {companies_updated} companies')
 
+@command
+async def update_pd_org_price_plans():
+    """
+    This patch sends a webhook to Pipedrive to update the price plan of all companies with a price plan
+    """
+    companies = await Company.filter(~Q(price_plan=None))
```
Comment (HenryTraill) [inline patch.py:100]: why does this need the ~Q filter? is this the only way to do this in hermes

Comment (PrenSJ2) [reply patch.py:100]: I believe so, as you cant do `price_plan__is_null=False`

Comment (PrenSJ2) [reply patch.py]: image

Comment (PrenSJ2) [reply patch.py]: can do `await Company.exclude(price_plan=None)`, yields same result

**Pull request merged**